docs/docs/guides/metadata.md

```diff
@@ -0,0 +1,106 @@
+#  Add metadata to your application
+
+This guide will walk you through the usage of Context Metadata in Keptn.
+
+After reading this guide you will be able to: 
+- add metadata to applications, workloads
+and tasks in two ways.
+- add those same information also in traces
+
+## Introduction
+
+`KeptnAppContext` is a custom resource in Keptn that allows you to add metadata
+and links to traces for a specific application.
+This enables you to enrich your Keptn resources and your traces with additional
+information, making it easier to understand and analyze
+the performance of your applications.
+
+Common usage of this feature may include:
+
+- adding commit id or references related to your GitOps/ CI-CD tooling
+- referring to different stages and actors 
+(ex. who committed this change? In what repo? What is the referred ticket? Was it a hot fix? )
+
+To exploit this feature you need to install Keptn and deploy an application,
+for instance, you can follow [step #1-#3 here](../getting-started/observability.md#step-1-install-keptn)
+
+To visualize traces you will require either Jaeger or Grafana:
+
+- Install
+  [Grafana](https://grafana.com/grafana/)
+  following the instructions in [Grafana Setup](https://grafana.com/docs/grafana/latest/setup-grafana/)
+  or the visualization tool of your choice.
+- Install
+  [Jaeger](https://www.jaegertracing.io/)
+  or a similar tool for traces following the instructions in
+  [Jaeger Setup](https://www.jaegertracing.io/docs/1.50/getting-started/).
+
+## Include metadata in workload traces
+
+To enrich the workload traces with custom metadata, you can use the
+`keptn.sh/metadata` annotation in your deployment.
+The values specified in the annotation
+are added as key-value attributes to the workload trace.
+
+Modify your deployment file adding`keptn.sh/metadata: "stage=dev"` annotation.
```

Comment:
```suggestion
Modify your Deployment file adding`keptn.sh/metadata: "stage=dev"` annotation.
```
Should this be "Deployment" yaml file?  And should we explain the meaning of `stage=dev` and what other values might be used?